src/test/regress/expected/citus_local_tables_queries_mx.out

```diff
@@ -683,23 +683,27 @@ SELECT count(*) FROM distributed_table WHERE b in
      0
 (1 row)
 
+SET citus.enable_ddl_propagation TO OFF;
```

Comment:
why do we need to disable ddl prop here? Shouldn't we able to support views on citus local table and dist tables?